#Goal: Find joint space velocities

#Inputs
float64[] q_pos

float64[] task_vel
#task_vel = x, y, z, wx, wy, wz
---
#Outputs
float64[] q_vel